new-app/tailwind.config.js

```diff
@@ -4,7 +4,13 @@ module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
   content: ["./src/**/*.{ts,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        "back-purple": "#4b296b",
```

Comment:
```suggestion
        "purple": "#4b296b",
```

you'll leave yourself a little more open for iteration if you keep contextual useage information out of the color names.